tiledb/sm/c_api/tiledb.cc

```diff
@@ -7226,6 +7264,26 @@ int32_t tiledb_serialize_array_max_buffer_sizes(
       ctx, array, subarray, serialize_type, buffer);
 }
 
+capi_return_t tiledb_deserialize_fragments_timestamps(
+    tiledb_ctx_t* ctx,
+    tiledb_serialization_type_t serialize_type,
+    const tiledb_buffer_t* buffer,
+    uint64_t* timestamp_start,
+    uint64_t* timestamp_end) noexcept {
+  return api_entry<tiledb::api::tiledb_deserialize_fragments_timestamps>(
```

Comment:
The wrapper can be `api_entry_context` which allows removing the `ctx` argument from the implementation function.